pkgs/desktops/elementary/pantheon-agent-geoclue2/default.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenv, fetchFromGitHub, elementary, pkgconfig, meson, ninja, vala, glib, gtk3
+, libgee, desktop-file-utils, dbus-glib, geoclue2, gobjectIntrospection, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "pantheon-agent-geoclue2";
+  version = "1.0";
+
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "elementary";
+    repo = pname;
+    rev = version;
+    sha256 = "0m7wbyqqivfwpd2m4jcxg4vl3vmfxkj7285h283p9m6nfh9vz960";
+  };
+
+  passthru = {
+    updateScript = elementary.updateScript {
+      repoName = pname;
+    };
+  };
+
+  nativeBuildInputs = [
+    desktop-file-utils
+    gobjectIntrospection
+    meson
+    ninja
+    pkgconfig
+    vala
+    wrapGAppsHook
+   ];
+
+  buildInputs = [
+    dbus-glib
```

Comment:
Could you open an issue about GDBus migration?

https://developer.gnome.org/gio/2.56/ch35.html

Reply:
@jtojnar 
This was an honest mistake apparently.
See https://github.com/elementary/pantheon-agent-geoclue2/commit/0b8939ff948120ed0c0746dac598e1cb3b487f6e
https://github.com/elementary/pantheon-agent-geoclue2/issues/24